### PR TITLE
Feature/issue 351

### LINKED ISSUE #351: StatisticalFactsheets: ForestArea

### LINKED ISSUE #356: StatisticalFactsheets: NaturallyRegeneratingForest

### PR DESCRIPTION
**Pull request by sorja**
Resolves #351 
Resolves #356

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -103,6 +104,7 @@
     "prop-types": "^15.7.2",
     "ramda": "^0.27.0",
     "react": "^16.13.1",
+    "react-chartjs-2": "^2.9.0",
```
Comment (minotogna) [inline package.json:107]: should we get rid of react-chartjs-2?

File: webapp/app/statisticalFactsheets/CarbonGrowingStock/CarbonGrowingStock.js [hunk 2]
```diff
@@ -8,14 +8,15 @@ const CarbonGrowingStock = (props) => {
...
+  const tableColumns = ['rowName', '1990', '2000', '2010', '2020']
+  const tableRows = ['carbon_stock_biomass_total', 'growing_stock_total', 'carbon_stock_total']
   const section = 'carbonAndGrowingStock'
 
   return (
     <div className="statistical-factsheets-carbon-growing-stock">
       <h3 className="header">{i18n.t(`statisticalFactsheets.${section}.title`)}</h3>
 
-      <Table tableHeads={tableHeads} section={section} levelIso={levelIso} />
+      <Table tableColumns={tableColumns} tableRows={tableRows} section={section} levelIso={levelIso} />
```
Comment (minotogna) [inline webapp/app/statisticalFactsheets/CarbonGrowingStock/CarbonGrowingStock.js:19]: should we call these properties just columns and rows?

File: webapp/app/statisticalFactsheets/ForestArea/ForestArea.js [hunk 3]
```diff
@@ -1,12 +1,39 @@
...
+  const chartHeads = ['1990', '2000', '2010', '2020']
+
+  const url = APIUtils.getUrl(levelIso)
+  const params = getParams(section)
+
+  const { data, dispatch: fetchData, loaded } = useGetRequest(url, {
+    params,
+  })
+
+  useEffect(fetchData, [url])
```
Comment (minotogna) [inline webapp/app/statisticalFactsheets/ForestArea/ForestArea.js:25]: this is somehow similar to all components. can we have a generic hook?

File: package.json (hunk 1 above)
Comment (sorja) [reply package.json:107]: Yes, let's do it after in #362

File: webapp/app/statisticalFactsheets/ForestArea/ForestArea.js (hunk 3 above)
Comment (sorja) [reply webapp/app/statisticalFactsheets/ForestArea/ForestArea.js:25]: I thought so too, will do

**Pull request merged**